chemprop/models/model.py

```diff
@@ -125,10 +125,14 @@ def fingerprint(
         return H if X_d is None else torch.cat((H, X_d), 1)
 
     def encoding(
-        self, bmg: BatchMolGraph, V_d: Tensor | None = None, X_d: Tensor | None = None
+        self,
+        bmg: BatchMolGraph,
+        V_d: Tensor | None = None,
+        X_d: Tensor | None = None,
+        i: int = None,
```

Comment:
is `None` a valid index value? I'm open to other opinions, but `-1` seems like a reasonable default index

Reply:
Good catch, `None` is not a valid value. The `-1` sounds like a good default value, and it will return the second-to-last block output.